rules/windows/defense_evasion_volume_shadow_copy_deletion_via_powershell.toml

```
@@ -0,0 +1,58 @@
+[metadata]
+creation_date = "2021/07/19"
+maturity = "production"
+updated_date = "2021/07/19"
+
+[rule]
+author = ["Austin Songer"]
+description = """
+Identifies use of WMI's Win32_ShadowCopy class or CIM cmdlets for shadow copy deletion on endpoints. This commonly occurs in tandem with ransomware or
+other destructive attacks.
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-endpoint.events.*", "logs-windows.*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Volume Shadow Copy Deletion via PowerShell"
+references = [
+    "https://docs.microsoft.com/en-us/previous-versions/windows/desktop/vsswmi/win32-shadowcopy",
+    "https://powershell.one/wmi/root/cimv2/win32_shadowcopy",
+    "https://www.fortinet.com/blog/threat-research/stomping-shadow-copies-a-second-look-into-deletion-methods",
+]
+risk_score = 73
+rule_id = "d99a037b-c8e2-47a5-97b9-170d076827c4"
+severity = "high"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Defense Evasion"]
+timestamp_override = "event.ingested"
+type = "eql"
+
+query = '''
+process where event.type in ("start", "process_started") and
+  process.name: ("powershell.exe", "pwsh.exe") and 
+  process.args : "Get-WmiObject" and process.args : ("*Win32_ShadowCopy*","*Delete()*", "*Disabled*", "Remove-WmiObject*") or
+  process.args : "Get-CimInstance" and process.args : ("*Win32_ShadowCopy*", "*Remove-CimInstance*") or
+  process.args : "gwmi" and process.args : ("*Win32_ShadowCopy*","*Remove-WmiObject*","*rwmi*") or
+  process.args : "gcim" and process.args : ("*Win32_ShadowCopy*","*rcim*")
```

```suggestion
process where event.type in ("start", "process_started") and
  process.name: ("powershell.exe", "pwsh.exe") and 
  process.args : ("*Get-WmiObject*", "*gwmi*", "*Get-CimInstance*", "*gcim*" ) and
  process.args : ("*Win32_ShadowCopy*") and
  process.args : ("*.Delete()*", "*Remove-WmiObject*", "*rwmi*", "*Remove-CimInstance*", "*rcim*" )
```

@austinsonger Let's replace the current query with this one